src/contracts/core/PaymentCoordinator.sol

```diff
@@ -0,0 +1,491 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
```

Comment:
Need to update pragmas!

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/514/commits/6f3d65e2dc454dcc0a3a57451e003455a69fa0fb
Good catch